parkingNum: 6
#resolution : 0.005
PL_SIZE: [0.8, 0.8]

# PL0: [ 1.0,  0.75]
PL0: [ 1.0,  0.75]

PL1: [ -1.0, 0.75]
PL2: [ 1.0,  -0.34]

PL3: [-1.0,  -0.34]
PL4: [ 1.0, -1.38]
PL5: [-1.0,  -1.38]

OutputSpot: [1.0, 1.55]
InputSpot: [-1.0, 1.55]

#OutputSpot: [1.05, 1.55]
#InputSpot: [-1.15, 1.55]
